.github/workflows/publish-image.yaml

```diff
@@ -3,6 +3,12 @@ name: Publish Images
 on:
   workflow_call:
     inputs:
+      ref:
+        type: string
+        default: main
+      repository: 
+        type: string
+        default: gingersnap-project/db-syncer
```

Comment:
The defaults here should be `${{ github.repository }}` and `${{ github.repository }}`.